src/components/Footer.astro

```diff
@@ -0,0 +1,65 @@
+<footer>
+  <section class="wrapper">
+    <div class="company">
+      <img src="img/Logo_black.svg" alt="Zasuń" class="footer_logo" />
+      <p>Piotr Zasuń</p>
+      <p>NIP: 626 269 96 59</p>
+    </div>
+    <div class="other">
+      <p>Copyright &copy 2023 Wszelkie prawa zastrzeżone</p>
+      <p>
+        Projekt i wykonanie: <a href="mailto:mateuszkarcz88@gmail.com"
+          >Unicorn Studio</a
+        >
+      </p>
+    </div>
+  </section>
+</footer>
+
+<style lang="scss">
+  footer {
+    margin-top: auto;
+    border-top: 1px solid var(--footer-color);
+
+    .wrapper {
+      margin: 40px 0;
+      gap: 16px;
+      display: block;
+    }
+
+    p {
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 1.5;
+      color: var(--footer-color);
+    }
```

Comment:
this looks like `.small-body` from `_typography.scss`

Reply:
Look at my combo

Reply:
Is it enough simply and clean? Good that you do not have a chance how it had looked before